### ISSUE TITLE
Conflict with the Sublime-Style-Column-Selection package

### ISSUE THREAD

**Issue by PhiLhoSoft** [tststs/atom-ternjs#247]
In the documentation, I see:

> Find definition (set your cursor position to one of variable, function or instance -> open context-menu and trigger "Find definition") or use the keybindings:
> 
>     alt+click (mac, windows, linux)

Also:

> Keybindings
> 
> List of keybindings. To use your own keybindings goto atom-ternjs package settings and disable keybindings.

BUT, Alt+click isn't a keybinding, it isn't in the above file.

BTW, for the record, to reach it we have to do "Open config folder" in the Settings, then navigate to packages > atom-ternjs > keymaps. Maybe it should be mentioned more explicitly for newbies like me... :smile: 

My question is: **Is there any mean to deactivate this shortcut?** (beside hacking the code...) Or to fix its behavior...

I use the [Sublime-Style-Column-Selection](https://atom.io/packages/Sublime-Style-Column-Selection) package which uses Alt+drag to do a column selection, and since a few releases of TernJS (or of this package), when I end my selection on an identifier, Atom-ternjs jumps to the definition of that identifier, which is very annoying and unwanted.

I see some solutions:

- The easy one: change the Sublime-Style-Column-Selection settings to use another modifier. But no, I use Alt+drag everywhere across the various editors I use to do a column selection, I don't want to change.
- Allow to deactivate this shortcut in atom-ternjs. I hope it is already possible, otherwise consider this issue to be a change request...
- The best solution: improve the Alt+click behavior, to work on a real click, not on a drag (here, mostly on a drag end, ie. a mouse up event). Perhaps by ensuring the mouse down and mouse up events happen on the same identifier...

Note that the last two solutions are complementary...

Thanks for your useful package, I would hate to renounce to it because of this glitch... :wink:

**Comment by tststs** [comment]
This should do the trick.

If it's not working for you after the next release, just reopen the issue and i'll choose another strategy.

I tried to avoid registering `mousemove`, `mouseup`and `mousedown`, calculating deltaX and deltaY and all that stuff cause it adds a lot of overhead for this simple problem. so this will just cancel the operation if there is any text selected in the current editor. Tested it with the mentioned package and it works for me.
